mkdocs.yml

```diff
@@ -96,6 +96,10 @@ nav:
                 - Job Key / Value Store: developers/bitesize/workflowExecution/jobKeyValueStore.md
```

Comment:
Is `Job key value store` and `Value Store` the same thing? If yes, then we should remove this entry as it is simply a placeholder

Reply:
Unfortunately not. The Job KV store is a way for jobs to record (eg) PAPI ids in the database in a backend agnostic way.

Reply:
Oh I see. Well then please ignore my comment :)